app/models/placements/schools/placements/build/placement.rb

```diff
@@ -87,4 +87,8 @@ def build_phase(phase)
     phase.presence ||
       (school.primary_or_secondary_only? ? school.phase : "Primary")
   end
+
+  def additional_subject_names
```

Comment:
Why is this only in the `build` version? Might as well be in the parent `Placement` model, and simplify the decorator

Reply:
Just a thought, you could try using this example in your controller to decorate the build version of a placement.
https://github.com/drapergem/draper#decorating-objects